src/app/api/system/reset/route.ts

```
@@ -0,0 +1,120 @@
+    }
+
+    // トランザクションでデータリセット実行
+    await prisma.$transaction(async (tx) => {
+      switch (resetType) {
+        case 'assignments':
+          const assignmentResult = await tx.assignment.deleteMany({})
+          deletedCounts.assignments = assignmentResult.count
+          break
+
+        case 'students':
+          const assignmentResult2 = await tx.assignment.deleteMany({})
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)